examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
ah just saw your guys' comments. So the full code is this:

```python
    if picks is None:
        picks = pick_types(inst.info, meg=True, eeg=True, ref_meg=False,
                           exclude='bads')
    if proj:
        # Copy first so it's not modified in place
        inst = inst.copy()
        inst.apply_proj()

    sfreq = inst.info['sfreq']
    if isinstance(inst, _BaseRaw):
        start, stop = np.where(time_mask)[0][[0, -1]]
        data, times = inst[picks, start:(stop + 1)]
```
So it's applying the projection and then taking the picks after that...it does sound like I could be messing up my indexing w/ a different set of picks